packages/runtime/test/src/interpreter/opcode-list.ts

```diff
@@ -7022,4 +7024,105 @@ describe("Teal Opcodes", function () {
 			expectRuntimeError(() => new Base64Decode([], 0), RUNTIME_ERRORS.TEAL.ASSERT_LENGTH);
 		});
 	});
+
+	describe("Tealv7: replace2 opcode", function () {
+		let stack: Stack<StackElem>;
+		this.beforeEach(() => {
+			stack = new Stack<StackElem>();
+		});
+
+		it("Should replace bytes correctly", () => {
+			const original = "0x11111111";
```

Comment:
You can use Uint8Array directly I think.

Reply:
yeah but I want to set up test like this to be look exactly from this github link
https://github.com/algorand/go-algorand/pull/4129/files#